qiskit_experiments/characterization/t2ramsey.py

```diff
@@ -80,7 +80,9 @@ def circuits(self, backend: Optional[Backend] = None) -> List[QuantumCircuit]:
             circ = qiskit.QuantumCircuit(1, 1)
             circ.h(0)
             circ.delay(delay, 0, self._unit)
-            circ.p(2 * np.pi * self._osc_freq, 0)
+            nosc = delay / self._delays[0]
```

Comment:
Will appreciate if someone with more experience with Ramsey experiments checks this line

Reply:
@nkanazawa1989 , can you have a look at this line? the origin is from Ignis. The idea is that the rotation in the phase gate increase in the same proportion as the delay.